ci/terraform/integration.tfvars

```diff
@@ -11,6 +11,7 @@ support_authorize_controller  = "1"
 support_2fa_b4_password_reset = "1"
 support_2hr_lockout           = "1"
 support_reauthentication      = "1"
+no_photo_id_contact_forms     = "1"
```

Comment:
please could you leave switched off in integration?

Reply:
done